### PR TITLE
optimize: update source header

### PR DESCRIPTION
**Pull request by slievrly**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did

update source header

### CODE REVIEW

**Review by leizhiyuan — APPROVED** [review]
LGTM

File: .licenserc.yaml [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
   license:
     spdx-id: Apache-2.0
     copyright-owner: Seata.io
     content: |
-      Copyright 1999-2019 Seata.io Group.
-    
-      Licensed under the Apache License, Version 2.0 (the "License");
-      you may not use this file except in compliance with the License.
-      You may obtain a copy of the License at
+      Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (funky-eyes) [inline .licenserc.yaml:23]: copyright-owner: Seata.io  Is that right?

Comment (slievrly) [reply .licenserc.yaml:23]: thanks for review, I have fixed it.

**Review by slievrly — COMMENTED** [review]
Evaluate the impact of the post-merge and open PRs. From what we can see, there shouldn't be any conflicts. Newly added files will require header modifications; otherwise, the license header might not pass. Additionally, the license header is not strictly enforced, which theoretically means it could still be merged even if it doesn't pass.

**Review by funky-eyes — APPROVED** [review]
LGTM

**Pull request merged**